src/components/ToDoList.js

```diff
@@ -0,0 +1,21 @@
+import "./ToDo.css";
+import ToDo from "./ToDo.js";
+
+function ToDoList({ toDo, onTogglePending, onDeleteToDo }) {
+  function renderToDos() {
+    return toDo.map((todoinput, index) => {
```

Comment:
Same here, i'd rename `todoinput`to just `todo` or `todoItem`, for the same reasons I expaleined before.